### PR TITLE
Added nautical miles option to polyline

### PR DESCRIPTION
**Pull request by konoamu**
Added this feature to allow the option to use nautical miles.
Allows leaflet-measurecontrol to display tool-tip unit conversion.
conversion

### CODE REVIEW

**Comment by edg2s** [comment]
There are now two binary flags used to select between three units: metres/miles/nautical. isMetric should be deprecated in favour of a 'units' option, which can take one of three values (and possibly more in the future).

File: dist/leaflet.draw-src.js [hunk 1]
```diff
@@ -214,6 +214,7 @@ L.Draw.Polyline = L.Draw.Feature.extend({
 			clickable: true
 		},
 		metric: true, // Whether to use the metric meaurement system or imperial
+		nautical: false, //Whether to use nautical miles
```
Comment (edg2s) [inline dist/leaflet.draw-src.js]: Space after //

File: dist/leaflet.draw-src.js [hunk 2]
```diff
@@ -1809,28 +1810,29 @@ L.GeometryUtil = L.extend(L.GeometryUtil || {}, {
...
+            var distanceStr;
+            if (isMetric) {
+                // show metres when distance is < 1km, then show km
+				if (distance > 1000) {
+					distanceStr = (distance / 1000).toFixed(2) + ' km';
+				} else {
+                    distanceStr = Math.ceil(distance) + ' m';
+                }
+            } else if (isNmi) {
+                distance *= 0.00053996; // nautical miles in 1 meter
```
Comment (edg2s) [inline dist/leaflet.draw-src.js]: distance /= 1852

would be clearer

**Comment by ddproxy** [comment]
This has been resolved in another PR. Thanks!

**Pull request closed without merging**